cinema/serializers.py

```diff
@@ -62,6 +99,7 @@ class Meta:
 
 class MovieSessionListSerializer(MovieSessionSerializer):
     movie_title = serializers.CharField(source="movie.title", read_only=True)
+    movie_image = serializers.ImageField(source="movie.image", read_only=True)
```

Comment:
According to the checklist, we should not use `serializers.CharField` for an image. You should use `serializers.ImageField` instead. Here is the correct way: `movie_image = serializers.ImageField(source="movie.image", read_only=True)`

Reply:
Hey, it is for the title. For image I do use the serializers.ImageField

Reply:
I think that is what I have.